### ISSUE TITLE
./lb delete removes similarly named posts from blog index

### ISSUE THREAD

**Issue by flippyy** [LukeSmithxyz/lb#6]
I had three test pages, test-post.html, test-post-2.html, and test-post-3.html. When I run`./lb delete` it removes all three pages from the blog index. I checked this again by making 3 new pages, yeet.html, yeet-2.html, and test.html. When I delete yeet.html using `./lb delete` it also removes yeet-2.html from the index, but test.html remains. The pages aren't being deleted, they're just no longer shown on the rolling blog page.

**Comment by LukeSmithxyz** [comment]
Do you have the most recent version? I added semi-duplicate handling recently. I just tried it myself and didn't get this problem.

**Comment by flippyy** [comment]
> Do you have the most recent version? I added semi-duplicate handling recently. I just tried it myself and didn't get this problem.

Yeah, I cloned it a few hours ago. The only change I've made to it that could've caused this is that I removed the `<++>` tag from the new post page, as I couldn't figure out what it does and it appeared in front of the date in pages.

**Comment by LukeSmithxyz** [comment]
Okay, I think I figured out the problem and fixed it. Too liberal matching for deletion. Want to try it again with the newest update?

**Comment by LukeSmithxyz** [comment]
Okay, great. Thanks for bring this to my attention.
